packages/core/etc/components-core.api.md

```diff
@@ -319,6 +319,21 @@ export interface ParticipantMedia<T extends Participant = Participant> {
 // @public (undocumented)
 export function participantPermissionObserver(participant: Participant): Observable<ParticipantPermission | undefined>;
 
+// @public (undocumented)
+export const participantTrackEvents: ParticipantEvent[];
+
+// Warning: (ae-forgotten-export) The symbol "RequireAtLeastOne" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type ParticipantTrackIdentifier = RequireAtLeastOne<{

```

Comment:
Personally, I find this type rather confusing and not too easy to understand. I'm hesitant about making this part of the public API.

Reply:
we can get rid of it, if we introduce the `byName` hooks instead. Does that sound good to you?

Reply:
this type is now only exposed on the core, not on the reacts package